doc/vxlan/EVPN/EVPN_VxLAN_Multihoming.md

```diff
@@ -0,0 +1,1639 @@
+| 1.0  | May-2024   | Syed Hasan Naqvi (Broadcom), Patrice Brissette (Cisco), Praveen Elagala (Broadcom), Rajesh Sankaran (Broadcom), Mike Mallin (Cisco), Kishore Kunal (Broadcom), Naveen Gamini (Cisco), Tapraj Singh (Cisco)   | Merged version   |
+
+<a id="About-this-Manual"></a>
+# About this Manual
+This document provides general information of EVPN VxLAN Multihoming feature implementation in SONiC.
+
+<a id="Scope"></a>
+# Scope
+This document describes the high level design and changes required for supporting Multihoming in EVPN VxLAN networks.
+This feature will provide [RFC 7432](https://datatracker.ietf.org/doc/html/rfc7432) and [RFC 8365](https://datatracker.ietf.org/doc/html/rfc8365) based Multihoming solution in SONiC.
+The ICCPd based MCLAG continue to co-exist as an alternate solution.
+
+<a id="Definition/Abbreviation"></a>
+# Definition/Abbreviation
+
+## Table 1: Abbreviations
+
+| Term     | Meaning                                   |
+| -------- | ----------------------------------------- |
+| (N)DF    | (Non-)Designated Forwarder                |
+| IRB      | Integrated Routing and Bridging           |
+| LVTEP    | Logical VTEP                              |
+| L3VNI    | (VxLAN) Layer3 Virtual Network Identifier |
+| MHD, SHD | Multi-Homed Device, Single-Homed Device   |
+| MCLAG    | Multi-Chassis LAG                         |
+| NHG      | Next Hop Group                            |
+| PIP, VIP | Primary IP Address, Virtual IP Address    |
+| SHG      | Split Horizon Group                       |
+| SHL      | Split Horizon List (List of VTEPs)        |
+| SVI      | Switch Virtual Interface                  |
+| VNI      | Virtual Network Identifier                |
+| VTEP     | VXLAN Tunnel End point                    |
+
+<a id="1-Feature-Overview"></a>
+# 1 Feature Overview
+
+<a id="11-Requirements"></a>
+## 1.1 Requirements
+
+<a id="11-Requirements"></a>
+### 1.1.1 Functional Requirements
+The following are functional requirements for EVPN VxLAN Multihoming:
+
+1. Support EVPN based all-active access redundancy.
+2. Support all of the routing and switching traffic flows currently supported on legacy MCLAG.
+3. Support following features on EVPN MH:
+	- Static Anycast Gateway
+	- BGP-EVPN Multihomed Ethernet-Segment
+	- ARP/ND suppression
+
+4. Support for Type-0 (Operator configured), Type-1 (LACP based), and Type-3 (System-MAC config based) Ethernet Segment IDs types.
+5. SONiC currently supports EVPN Vlan-based service. No new service-types are introduced with this feature
+
+The following are non-goals for the EVPN Multihoming feature:
+
+1. Support of single-active, port-active or single-flow-active EVPN services
+2. Support for DF election modes besides preference-based
+3. Asymmetric IRB support
+4. Support of VRRP over EVPN Multihoming. Static Anycast Gateway should be used instead.
+5. EVPN Multihomed interface as router-port or routed sub-interface. EVPN Multihomed interface can only be a switchport.
+
```

Comment:
Are there any Interop requirements between EVPN MH and MCLAG based multi-homing solutions?

Reply:
There are mutually exclusive

Reply:
They are mutually exclusive within the same node. But my question is more about the interop between Leaf1 (EVPN-MH) to Leaf2A-Leaf2B (MLAG pair)

Reply:
That is correct. MC-LAG and EVPN-MH are using different mechanism to exchange information (ICCP vs BGP).